crates/fluvio-smartengine/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-smartengine"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
since API is changed, we need to bump this to 0.5.0

Reply:
done